cocotb/share/lib/verilator/verilator.cpp

```diff
@@ -84,5 +84,11 @@ int main(int argc, char** argv) {
     tfp->close();
 #endif
 
+// VM_COVERAGE is a define which is set if Verilator is 
```

Comment:
The whitespace at the end of this line (and maybe more such lines) makes CI fail.